### ISSUE TITLE
keep up with node versions?

### ISSUE THREAD

**Issue by eklem** [fergiemcdowall/search-index#435]
Should we make a plan on how to support the [current versions of nodejs](https://github.com/nodejs/Release)?

Now that is 4, 6, 8, 9 and soon 10, and we test on [4, 6 and 7](https://travis-ci.org/fergiemcdowall/search-index).

![Node Release schedule and supported versions](https://github.com/nodejs/Release/blob/master/schedule.png?raw=true)

**Comment by eklem** [comment]
So, then in April we need to support 12 as well? And then it's also the official end of life for v6, even if people use it longer.

[NodeJS Releases](https://nodejs.org/en/about/releases/)

**Comment by fergiemcdowall** [comment]
Yes- I'm actually pretty sure that it works on 12 (and if not, it will be worth getting it to work)
